src/taco-cli/cli/utils/cordovaWrapper.ts

```diff
@@ -188,6 +190,24 @@ class CordovaWrapper {
         options: { logLevel?: tacoUtility.InstallLogLevel, isSilent?: boolean, captureOutput?: boolean } = {}): Q.Promise<T | string> {
         return CordovaHelper.tryInvokeCordova<T | string>(apiFunction, () => CordovaWrapper.cli(processArgs(), options.captureOutput), options);
     }
+
+    /**
+     * Returns a path to the local Cordova command if ran from a taco project, otherwise it will return the
+     * global Cordova command.
+     */
+    private static getCordovaCommand(): Q.Promise < string > {
```

Comment:
Maybe rename to getCordovaPath? My first reaction to the name was that it would return a command intended for cordova, rather than returning which cordova should be spawned.